isaric/parsers/ccp-guinea.toml

```diff
@@ -680,9 +690,14 @@
   ]
 
 [visit.treatment_oxygen_therapy]
-  field = "overall_oxygen_cmoccur"
-  description = "Oxygen therapy?"
-  ref = "Y/N/NK"
+  combinedType = "any"
+  fields = [
+    { field = "overall_oxygen_cmoccur", description = "Oxygen therapy?", ref = "Y/N/NK" },
+    { fieldPattern = "overall_oxygen_cmtype___.*", description = "Oxygen interface type", values = { 2 = true } },
+    { field = "overall_invasive_proccur", description = "Invasive ventilation?", ref = "Y/N/NK" },
+    { field = "overall_noninvasive_proccur", description = "Non-invasive ventilation?", ref = "Y/N/NK" },
+    { field = "overall_extracorp_prtrt", description = "Extracorporeal (ECMO) support", ref = "Y/N/NK" },
+  ]
 
```

Comment:
@pipliggins the ICU variables can_oxy_ther, non_inv_vent, inv_vent and ecls can be added but not much data provided for these (overall_ variables not completed)